modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/DatabaseRegistry.java

```diff
@@ -308,12 +308,12 @@ void retrieveAndUpdateDatabase(String databaseName, GeoIpTaskState.Metadata meta
 
     void updateDatabase(String databaseFileName, String recordedMd5, Path file) {
         try {
-            LOGGER.info("database file changed [{}], reload database...", file);
```

Comment:
Do you think we should keep this message at `DEBUG` just in case it helps us investigate problems where stuff _does_ get stuck or go slow during the loading?